### DISCUSSION TITLE
Permanently disable the constantly scrolling text (about drag and dropping tabs)?

### DISCUSSION THREAD

**Discussion by stderr-mila** [piroor/treestyletab#3460] [Q&A]
Hello is there a way to permanently disable the scrolling text at the bottom of TSS sidebar?

![Screenshot from 2024-02-19 11-05-39](https://github.com/piroor/treestyletab/assets/62907702/ffa95b4f-41f0-443b-87f0-77ee8b63bfc3)

**Comment by piroor** [comment]
There is the option under the "Drag and Drop" section.
![image](https://github.com/piroor/treestyletab/assets/70062/7c2b8c54-c541-41f7-a757-dfee767fcb4a)
